src/pages/profile/profile.tsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+// import { useStore } from 'effector-react';
```

Comment:
use index.ts in pages/profile and import from there

Reply:
fixed